### PR TITLE
theme: introduce cinnamon

### PR DESCRIPTION
**Pull request by JanDeDobbeleer**
### Prerequisites

- [x] I have read and understand the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

### Description

[Description of the change]

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

**Comment by JanDeDobbeleer** [comment]
@cinnamon-msft I'm going to need a review here ;-)

File: themes/cinnamon.omp.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      "alignment": "left",
+      "segments": [
+        {
+          "type": "root",
+          "style": "powerline",
+          "powerline_symbol": "\uE0B0",
+          "foreground": "#100e23",
+          "background": "#ffe9aa",
+          "properties": {
+            "root_icon": "\uFA90"
```
Comment (cinnamon-msft) [inline themes/cinnamon.omp.json]:
```suggestion
            "root_icon": "\uF451"
```
I meant to make this one the crown (👑) after changing back from a space theme I made. :)

Comment (JanDeDobbeleer) [reply themes/cinnamon.omp.json]: done

**Comment by JanDeDobbeleer** [comment]
@cinnamon-msft fixed. I wonder if the emojis will work fine. No experience with that myself.

**Comment by cinnamon-msft** [comment]
I think it depends on the terminal emulator. Not sure if all emulators will display them.

**Pull request merged**